### PR TITLE
update test paths and add test_bedgraph

### PR DESCRIPTION
**Pull request by yuukiiwa**
https://github.com/nf-core/modules/pull/371#issue-599698345
`ucsc_begraphtobigwig`, `ucsc_bed12tobigbed`, and `minimap2/align` pass all the checks.
For `bowtie/align`, `bowtie/build`, and `stringtie2`, I have removed their md5sum, but the error say that they can't find the outputfiles.

### CODE REVIEW

File: tests/config/test_data.config [hunk 1]
```diff
@@ -1,70 +1,71 @@
-// Base directory for test data
-def test_data_dir = "${launchDir}/tests/data/"
-
```
Comment (drpatelh) [inline tests/config/test_data.config:3]: Oops what happened here? Can we revert this change if the file hasn't changed?

Comment (drpatelh) [reply tests/config/test_data.config:3]: Ah, you added the bedgraph file. Wonder why it is not just showing that change. Did you copy and paste the entire file contents?

**Review by drpatelh — COMMENTED** [review]
Amazing! Thanks @yuukiiwa ! It seems like you have deleted ALL of the md5sum checks? Are you able to try to add some of them in where appropriate?

File: tests/config/test_data.config (hunk 1 above)
Comment (yuukiiwa) [reply tests/config/test_data.config:3]: Hi @drpatelh, I got conflicts merging the GoekeLab/module-master branch to nf-core/module-master branch, and I resolved it on the github resolve conflict page:
Screenshot 2021-03-26 at 8 59 54 AM

**Comment by drpatelh** [comment]
Will be fixed in https://github.com/nf-core/modules/pull/420. Thanks @yuukiiwa !
